x-pack/legacy/plugins/maps/public/angular/get_initial_layers.js

```diff
@@ -27,11 +27,7 @@ export function getInitialLayers(savedMapLayerListJSON, isDarkMode) {
 
   const isEmsEnabled = chrome.getInjected('isEmsEnabled', true);
   if (isEmsEnabled) {
```

Comment:
nice simplification. with auto-mode, this can just happen automagic now at layer-add time